assets/js/atomic/blocks/product-elements/stock-indicator/style.scss

```diff
@@ -3,15 +3,4 @@
 	margin-bottom: em($gap-small);
 	display: block;
 	@include font-size(small);
-
-	&--in-stock {
-		color: $in-stock-color;
-	}
-	&--out-of-stock {
-		color: $no-stock-color;
-	}
-	&--low-stock,
-	&--available-on-backorder {
-		color: $low-stock-color;
-	}
```

Comment:
We are removing the feature that store owners actually need here. The current Supports API doesn't allow multiple colors, so I don't think it's a good idea to remove styles for stock statuses. Instead, we should accept the limitation of the Supports API. In the other words, ignoring the color, only add global style support for typography.

We can also create custom inspector settings to control colors for each state, and it will be local block style only, not global style.

Reply:
> We are removing the feature that store owners actually need here. The current Supports API doesn't allow multiple colors, so I don't think it's a good idea to remove styles for stock statuses. 

Before proceeding, I asked the rest of the team. Please check this discussion (p1641398990217000-slack-C02FL3X7KR6), maybe we can continue the discussion there.